src/osd/scheduler/mClockScheduler.h

```diff
@@ -189,12 +207,34 @@ class mClockScheduler : public OpScheduler, md_config_obs_t {
   // Enqueue the op in the front of the regular queue
   void enqueue_front(OpSchedulerItem &&item) final;
 
+  void enqueue_strict(unsigned prio, OpSchedulerItem &&item);
```

Comment:
Add appropriate comment for the enqueue_* methods and other places where new methods are introduced.